### ISSUE TITLE
document reduction in stepper current, motors get very hot when idle

### ISSUE THREAD

**Issue by daduke** [OpenScanEu/OpenScan-PCB#10]
Hi,

I noticed that both steppers get very hot when idle. Vref on the A4988s was > 0.9 V, which I'm sure isn't necessary for this scanner. I reduced it to 0.5 V which is plenty, motors stay cool.

-Christian

**Comment by probably-Erwins-Cat** [comment]
I agree. The typical Vref is often too high and can be reduced to 0.5V when the rotor runs smoothly. Nonetheless, not all self-printed parts are of high dimensional accuracy and setting the Vref (and thus maximum torque of the motors) is up to the users. In the OpenScan discord is also some info about that.

**Comment by tennysog** [comment]
@daduke Thank you for your post on this, I've noted my PLA platforms deforming due to the heat involved and will try this fix.

For the next rev of the PCB package that's in the works something like TI's DRV8880 could be used to drive the stepper motors instead of A4988s and add traces and software to utilize the pins to let users set the current control from the GUI in steps of 25% total Vref

**Comment by probably-Erwins-Cat** [comment]
Indeed, the A4988 are somewhat outdated drivers. The next PCB will introduce TMC drivers, probably 2208 or similar generation. They provide the same torque at much lower heat dissipation, have better cooling fins, and some other goodies, such as automatic current reduction when idling.
I am uncertain about the current control idea via GPIO pins as the newer generation.
